sdc/tests/test_dataframe.py

```diff
@@ -1103,6 +1110,28 @@ def test_impl(df):
         df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n)**2})
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
 
+    @skip_sdc_jit
+    def test_isin_ser1(self):
+        def test_impl(df):
+            vals = pd.Series([2, 3, 4])
+            return df.isin(vals)
```

Comment:
Could we pass `vals` as a parameter to the `test_impl`? if it's so let's combine all the tests to a single one through `unittest.subTest`